addons/website/static/tests/builder/image_shape.test.js

```diff
@@ -3,10 +3,154 @@ import { animationFrame, queryFirst, waitFor } from "@odoo/hoot-dom";
 import { contains } from "@web/../tests/web_test_helpers";
 import { defineWebsiteModels, setupWebsiteBuilder } from "./website_helpers";
 import { delay } from "@web/core/utils/concurrency";
+import { imageShapeDefinitions } from "@html_builder/plugins/image/image_shapes_definition";
 import { testImg } from "./image_test_helpers";
 
 defineWebsiteModels();
 
+const testImgWithShapeAndClass = `
+<img src='/web/image/website.s_text_image_default_image'
+    data-attachment-id="1" data-original-id="1"
+    data-original-src="/website/static/src/img/snippets_demo/s_text_image.webp"
+    data-mimetype-before-conversion="image/webp"
+    data-mimetype="image/webp"
+    data-shape="html_builder/geometric/geo_shuriken"
+    data-file-name="s_text_image.webp"
+    class="o_class_one o_class_two"
+    >
+`;
+const testImgShapeWithOnlyClass = `
+<img src='/web/image/website.s_text_image_default_image'
+    class="o_class_one o_class_two"
+    >
+`;
+const testImgShapeWithOnlyShape = `
+<img src='/web/image/website.s_text_image_default_image'
+    data-shape="html_builder/geometric/geo_shuriken"
+    >
+`;
+test("Applying shape with imageShapeClass should not generate an SVG and clear border radius", async () => {
+    // Update geo_shuriken to apply shape through imageShapeClass
+    imageShapeDefinitions.basic.subgroups.geometrics.shapes[
```

Comment:
Maybe `patchWithCleanup` could be usefull here. Also, If you want to do a modification of `imageShapeDefinitions` before few tests, you can add those tests in a `describe` and use `beforeEach` inside the describe :+1: